### PR TITLE
update doc about enable rbac on local cluster

### PR DESCRIPTION
**Pull request by xilabao**
reference to https://github.com/kubernetes/kubernetes/pull/34619

### CODE REVIEW

File: contributors/devel/local-cluster/local.md [hunk 1]
```diff
@@ -30,6 +30,10 @@ and run the following (since one needs sudo access to start/stop Kubernetes daem
 cd kubernetes
 hack/local-up-cluster.sh
 ```
+If you want to enable RBAC on local cluster, you can run the following:
+```shell
+ALLOW_ANY_TOKEN=true ENABLE_RBAC=true hack/local-up-cluster.sh
```
Comment (liggitt) [inline contributors/devel/local-cluster/local.md]: The ALLOW_ANY_TOKEN part isn't related to RBAC

**Comment by xilabao** [comment]
@liggitt done. PTAL

**Comment by philips** [comment]
LGTM.

cc @ericchiang

**Comment by bgrant0607** [comment]
cc @deads2k @pweil-

**Pull request merged**